test/midje/emission/t_api.clj

```diff
@@ -7,20 +7,6 @@
             [midje.emission.plugins.test-support :as plugin]
             [midje.config :as config]))
 
-(fact load-plugin
-  (fact "symbols are required"
-    (emit/load-plugin 'symbol) => irrelevant
-    (provided
-      (require 'symbol :reload) => anything))
-
-  (fact "strings are given to load-file"
-    (emit/load-plugin "string") => irrelevant
-    (provided
-      (load-file "string") => anything)))
```

Comment:
removing these tests because they test simple `if` branch logic that dispatches to `clojure.core` functions, so it becomes hard to meaningfully test after the updates of this PR